docker/erobs-common-img/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM osrf/ros:humble-desktop-full
+
+# Set display
+ENV DISPLAY :1
+
+RUN apt-get update \
+        && apt-get -y install \
+        python3-pip \
+        ros-${ROS_DISTRO}-rclpy \
+        ros-${ROS_DISTRO}-ur \
+        build-essential \
+        python3-colcon-common-extensions
+RUN apt-get install -y x11vnc xvfb 
+# Clean up
+RUN apt-get autoremove -y \
+        && apt-get clean -y \
+        && rm -rf /var/lib/apt/lists/*
+RUN pip3 install rosdep && \
+        rosdep update
+
+RUN mkdir /root/ws/src -p
+
+WORKDIR /root/ws/src
+
+# TODO @ChandimaFernando. Revert back after tests. RUN git clone https://github.com/maffettone/erobs.git -b humble
+RUN git clone https://github.com/ChandimaFernando/erobs.git -b humble_hande_control
+RUN git clone https://github.com/PickNikRobotics/serial-release.git -b release/humble/serial
+RUN git clone https://github.com/PickNikRobotics/ros2_robotiq_gripper.git -b cmake-external-project-serial
```

Comment:
Can we make this work with the `setup.sh` and `build.sh` files?

Reply:
Dockerfile now points to setup.sh and build.sh
https://github.com/maffettone/erobs/blob/be77ef51c7707cbda785d3a82a71f96ec5b16b16/docker/erobs-common-img/Dockerfile#L28-L32

Reply:
Note: are these examples needed ?
https://github.com/maffettone/erobs/blob/be77ef51c7707cbda785d3a82a71f96ec5b16b16/src/ros2.repos#L7-L10
Building them increases build-time. I can ignore them one by one as build arguments, but I prefer not to copy them to the workspace at all.

Reply:
No we can drop.